src/components/navigation/Navbar.js

```diff
@@ -33,6 +35,13 @@ const Navbar = () => {
               style={{ textDecoration: 'none' }}
             >
               Avocadoro
+              <Icon>
+                <img
+                  src={AvocadoroIcon}
+                  alt="AvocadoroToast"
+                  height={20}
```

Comment:
Magic number, prefer using const.

Perhaps you can create a new file `src/const/ui.js` to store all ui const?

Reply:
ok boss